### PR TITLE
Reattach processes started by the tray launcher when restarted

### PR DESCRIPTION
**Pull request by danhuaxu**
Addressing issue #67.

1. Tray launcher is now able to reattach processes started by it after the tray launcher itself is restarted.
2. If the process is removed when tray launcher is offline, this process will not show up when the tray launcher restarts.
3. Uses creation time to distinguish processes with the same pid. (Not tested because I don't know how to create a process with a given pid)

This PR also fixes issue #32.

### CODE REVIEW

File: src/tray_launcher/child_script.py [hunk 1]
```diff
@@ -5,32 +5,47 @@
...
-        self.current_PIDs = []
+    def __init__(self, pid, create_time, script_path, logging_log):
+        """Create a ChildScript instance, used in reattaching processes when the tray launcher restarts.
+
+        Args:
+            pid: int
+            create_time: float
+            script_path: Path
+            logging_log: Path
+        """
```
Comment (Samuel-Gebe) [inline src/tray_launcher/child_script.py:22]: Can you add more description on what each argument is. For example, "create_time: float, time of when script was initialized in [seconds?]"

**Comment by Samuel-Gebe** [comment]
@danhuaxu I left some comments. Let me know if you have any questions.

File: src/tray_launcher/cli.py [hunk 2]
```diff
@@ -73,6 +73,8 @@ def test(self, data):
 
     def start(self, data):
         """Process the "start" command. Start new scripts."""
+        self.gui.check_active_processes()
+
         for path_str in data[1:]:
             file_path = self.gui.to_loaded_path(Path(path_str))
             if file_path is not None:
```
[the remark this answers is not in the document]
Comment (danhuaxu) [reply src/tray_launcher/cli.py:80]: No I don't.

**Comment by danhuaxu** [comment]
@Samuel-Gebe Added some more docstrings.

File: src/tray_launcher/child_script.py [hunk 3]
```diff
@@ -5,32 +5,47 @@
 import time as _t
 from pathlib import Path
 
+import psutil as _ps
```
Comment (fanmingyu212) [inline src/tray_launcher/child_script.py:8]: add this to requirements.

**Review by fanmingyu212 — COMMENTED** [review]
Only one comment on adding `psutil` to requirements. I tested it on my local computer and it works great!

File: src/tray_launcher/child_script.py (hunk 3 above)
Comment (danhuaxu) [reply src/tray_launcher/child_script.py:8]: Added. Thank you!

**Review by fanmingyu212 — APPROVED** [review]
lgtm

**Comment by danhuaxu** [comment]
I will update to PyPi later.

**Pull request merged**